### PR TITLE
Chore/logging

### PR DESCRIPTION
**Pull request by donnapep**

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees Please review. Thx.

**Comment by donnapep** [comment]
@stulees I also rebased against master. Thx.

File: test/unit/widget/video-spec.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      "company_id": '"companyId"',
+      "display_id":'"displayId"'
+    };
+
+    RiseVision.Video.logEvent({ "event": "ready" });
+
+    expect(logSpy).to.have.been.calledWith("video_events", params);
+  });
+
+  it("should call spy with correct parameters when only the event details parameter is set", function() {
```
Comment (stulees) [inline test/unit/widget/video-spec.js]: @donnapep Currently our "video_events" table has the `event` field as required. If it's omitted in an insert, it should likely result in an error. Unless we don't make that field required, `event` always has to be submitted.

Comment (donnapep) [inline test/unit/widget/video-spec.js]: @stulees OK, but I'm not sure what needs to change with this code. Although I have this test, I'm not actually logging any events without an event name. The logic for no event name is already enforced in logger.js, so do we need to enforce the same rule here?

Comment (stulees) [inline test/unit/widget/video-spec.js]: @donnapep Okay, I guess this test is somewhat confusing then. I understood it as the `event` isn't required. Do we need this test?

Comment (donnapep) [inline test/unit/widget/video-spec.js]: @stulees Guess not. I removed it. Thx.

**Comment by stulees** [comment]
@donnapep :+1:

**Pull request merged**